src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,63 @@
+package duke.command;
+
+import duke.interaction.Ui;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.util.Storage;
+
+import java.util.ArrayList;
+
+/**
+ * Represents the Command for the "add" input by the user.
+ * It adds a new task to the task list.
+ *
+ * @author  Hardy Shein
+ * @version 0.1
+ */
+public class AddCommand extends Command {
+
+    private String toAdd;
+    private Task.TaskType taskType;
+
+    public AddCommand() {
+
+    }
+
+    /**
+     * AddCommand constructor.
+     * @param toAdd string name of new task.
+     * @param taskType of the new task.
+     */
+    public AddCommand(String toAdd, Task.TaskType taskType) {
+        this.toAdd = toAdd;
+        this.taskType = taskType;
+    }
+
+    /**
+     * Executes Add behaviour of adding a new task.
```

Comment:
Hi there, do remember to leave a blank line after the description and the "@" tag as per the CS2103T coding standards.